.env.test

```
@@ -84,6 +84,7 @@ CORS_ALLOW_ORIGIN='^https?://localhost(:[0-9]+)?$'
 ###> symfony/messenger ###
 # Choose one of the transports below
 MESSENGER_TRANSPORT_DSN=in-memory://
+MESSENGER_ROSTERING_FILE_UPLOADED_TRANSPORT_DSN=in-memory://
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MESSENGER_ROSTERING_FILE_UPLOADED_TRANSPORT_DSN key should go before the MESSENGER_TRANSPORT_DSN key

(UnorderedKey)